zinc/src/sbt-test/source-dependencies/patMat-scope/src/main/scala/foo/Bar.scala

```diff
@@ -0,0 +1,6 @@
+package foo
+
+// This class is used to pad the number of source code.
```

Comment:
@jvican Plz see the commit message. Basically it works on my machine, but on Drone I am seeing:

```
[debug] Recompiling all 3 sources: invalidated sources (2) exceeded 50.0% of all sources
```

Reply:
KOh wow, unexpected. I opened a ticket so that we can keep track of this one in the future. https://github.com/sbt/zinc/issues/430

Reply:
To disable this heuristic, we can use this:
https://github.com/sbt/zinc/blob/3a64b8aefab45dac661056cc6d0a018a953bf4e2/zinc/src/sbt-test/source-dependencies/specify-inc-options/incOptions.properties#L2